research/1L.md

```diff
@@ -1,17 +1,22 @@
 # Label: 1L
 
 ## Description
-
+So far, there are at least 4 variants
 
 ### Carriers observed on
 
 This message label has been observed on the following carriers.
 
-Carrier | IATA Code | ICAO Code
+Carrier | IATA Code | ICAO Code |
```

Comment:
Is this pipe needed?

Reply:
Ohhhh, good catch... originally I added another column but decided against it. Must've forgotten to remove it

Reply:
Just proof that I actually look at everything people contribute :)